.travis.yml

```diff
@@ -0,0 +1,219 @@
+conditions: v1
+sudo: false
+
+cache:
+  pip: true
+  directories:
+  - $HOME/.cache/pre-commit
+  - $HOME/.pre-commit
+  - $HOME/virtualenv/python$(python -c 'import platform; print(platform.python_version())')
+  - $HOME/Library/Caches/Homebrew
+
+services:
+- docker
+
+language: python
+
+.fixtures:  # is not used by Travis CI, but helps avoid duplication
+- &if-cron-or-manual-run
+  if: type IN (cron, api)
+- &py-27
+  python: "2.7"
+- &py-36
+  python: "3.6"
+- &py-37
+  python: "3.7"
+  sudo: required
+  dist: xenial
+- &lint
+  <<: *py-36  # Because 3.7 requires a VM is Travis CI, which is "heavy"
```

Comment:
Define "heavy

Reply:
All other jobs use Docker containers, 3.7 requires having a very "special" config running in VM. Containers are usually faster, so I chose Python 3.6 over 3.7.